src/Pages/suyeonkim/Main/Feeds/Feed/Feed.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import FeedTop from './FeedTop/FeedTop';
+import FeedsBtn from './FeedBtn/FeedsBtn';
+import WhoLike from './WhoLike/WhoLike';
+import FeedImg from './FeedImg/FeedImg';
+import FeedComment from './FeedComment/FeedComment';
+
+class Feed extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      feedComment: [],
+      commentId: 0,
+      inputComment: '',
+    };
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/data/suyeonkim/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(commentLi => {
+        this.setState({
+          feedComment: commentLi,
+        });
+      });
+  }
+
+  addComment = e => {
+    e.preventDefault();
+    const { feedInfo } = this.props;
+    const matchId = this.state.feedComment.filter(
+      el => feedInfo.feedId === el.feedId
+    );
+
+    const list = [...this.state.feedComment];
+    this.setState({
+      feedComment: list.concat({
+        id: matchId.length + 1,
+        userName: 'suyeonKim',
+        content: this.state.inputComment,
+        isLiked: false,
+        feedId: feedInfo.feedId,
+      }),
+      inputComment: '',
+    });
+  };
+
+  temValue = e => {
+    this.setState({
+      inputComment: e.target.value,
+    });
+  };
+
+  render() {
+    const { feedInfo } = this.props;
+    const { feedComment } = this.state;
+
+    const list = feedComment.filter(el => el.feedId === feedInfo.feedId);
```

Comment:
```suggestion
    const list = feedComment.filter(comment => comment.feedId === feedInfo.feedId);
```
el, data보다 명시적인 이름을 써주는 습관 들여주세요